### ISSUE TITLE
[IDE] Debugger - XSJS files debugging is weird

### ISSUE THREAD

**Issue by ThuF** [SAP/xsk#1181]
Details about the issue could be found here: https://github.com/eclipse/dirigible/issues/1539

Inside XSK the experience is even worse, as there are 10+ modules _(the whole [XSJS API](https://github.com/SAP/xsk/blob/main/modules/api/api-xsjs/src/main/resources/META-INF/dirigible/xsk/api.js) and its dependencies)_ that you have to go through in order to debug the following hello world code:

```javascript
$.response.setBody("Hello World!");
```

**Comment by vmutafov** [comment]
This should be closed once we have a new release of the Dirigible.

**Comment by ThuF** [comment]
@vmutafov, @tdermendjiev is this issue closed by https://github.com/eclipse/dirigible/pull/1567?

**Comment by vmutafov** [comment]
@ThuF this is partially fixed. Although we have removed some unnecessary module loading related debug code, the issue is still observed as we treat XSK/Dirigible internal JS code as non-internal when loading it into the GraalJS. 
I have created a PoC about marking the xsjs APIs as internal but I'm still testing it in the XSK environment. So far it passes the Dirigible unit tests. I'll create issues for these improvements in the Dirigible and link them here.
